GOVERNANCE.md

```diff
@@ -28,8 +26,14 @@ No more than one-fourth of the CC members may be affiliated with the same employ
 
 The CC members shall consist of active members of Community Projects and the two Individual Membership Directors as defined in Section 10.
 
+The CC may, at its discretion, invite any number of non-voting observers to participate in the public portion of CC discussions and meetings.
```

Comment:
Using the term "The CC" here - not entirely sure what this would describe, since the CC is a group of people.

How would a member of the CC go about inviting a non-voting observer to participate in the public portion of the CC discussions/meetings? Would they need to create an issue on GitHub requesting or notifying of the invitation, or is it entirely open for any CC member to invite a non-voting observer at their discretion?